public/css/index.css

```diff
@@ -210,3 +210,7 @@ input, select{
         0% 10%
     )
 }
+
+.trash-drop-hover{
+    background-color: #3d0000 !important;
```

Comment:
Is this a CSS attribute or a comment? If it's a comment, why is this line important?

Reply:
This is an attribute. The `!important` causes the background color to overwrite all other background attributes on the same element, which is needed in this case to overwrite the jQuery background for trash. Its use is controversial, but this is a case where it's useful and won't lead to any complications.

Reply:
OK, got it. Add a comment noting the controversial nature of this particular attribute and why you still used it.